spec/services/command/__snapshots__/command.spec.ts.snap

```diff
@@ -1,5 +1,59 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
+exports[`deploy > has a usage 1`] = `
+"Deploy your Gadget application's development source code to production.
+
+USAGE
+  ggt deploy [DIRECTORY] [--app=<name>]
+
+ARGUMENTS
+  DIRECTORY         The directory to sync files to (default: \\".\\")
+
+FLAGS
+  -a, --app=<name>  The Gadget application to sync files to
```

Comment:
is this a flag for this command?

Reply:
The description should be changed! `"The Gadget application to deploy"`. If the user has never run a sync, running deploy first will prompt them to select an app from their available apps (and then will prompt them to do the first time sync and then they can continue to deploy). After that they don't need to pass in the `--app` flag.